index.js

```diff
@@ -11,43 +11,41 @@ const fs = require("fs")
 // Create a Secrets Manager client
 var secrets_client = new AWS.SecretsManager({
     region: region
-});
-try {
-  data = await secrets_client.getSecretValue({SecretId: secretName});
-  secret = JSON.parse(data.SecretString)
-} catch (err) {
-  if (err.code === 'DecryptionFailureException')
-      // Secrets Manager can't decrypt the protected secret text using the provided KMS key.
-      // Deal with the exception here, and/or rethrow at your discretion.
-      throw err;
-  else if (err.code === 'InternalServiceErrorException')
-      // An error occurred on the server side.
-      // Deal with the exception here, and/or rethrow at your discretion.
-      throw err;
-  else if (err.code === 'InvalidParameterException')
-      // You provided an invalid value for a parameter.
-      // Deal with the exception here, and/or rethrow at your discretion.
-      throw err;
-  else if (err.code === 'InvalidRequestException')
-      // You provided a parameter value that is not valid for the current state of the resource.
-      // Deal with the exception here, and/or rethrow at your discretion.
-      throw err;
-  else if (err.code === 'ResourceNotFoundException')
-      // We can't find the resource that you asked for.
-      // Deal with the exception here, and/or rethrow at your discretion.
-      throw err;
-}
-
 });
 //Expect payload on the event object
 exports.handler =  async function(event, context) {
   process.env[`PATH`] = process.env[`PATH`] + `:` + process.env[`LAMBDA_TASK_ROOT`]
-  //console.log("EVENT: \n" + JSON.stringify(event, null, 2))
-  //return context.logStreamName
+  try {
+    data = await secrets_client.getSecretValue({SecretId: secretName}).promise();
+    console.log("data", data)
+    console.log("secretString", data.SecretString)
```

Comment:
That should probably not make it into the final version :)

Reply:
ah, yeah

Reply:
removed